src/frontends/paddle/tests/places.cpp

```diff
@@ -11,49 +13,38 @@
 using namespace ov::frontend;
 
 const std::string model_file = std::string(TEST_PADDLE_MODELS_DIRNAME) + "place_test_model/place_test_model.pdmodel";
+const std::string vars_name_file = std::string(TEST_PADDLE_MODELS_DIRNAME) + "place_test_model/vars_name.txt";
```

Comment:
Have a try to change the path as below
```suggestion
FrontEndTestUtils::make_model_path(std::string(TEST_PADDLE_MODELS_DIRNAME) + "place_test_model/vars_name.txt");
```

Reply:
> Have a try to change the path as below

build failed. 

<img width="1374" alt="image" src="https://github.com/openvinotoolkit/openvino/assets/55493212/d604d2b5-e916-43ee-b40f-ed7f585392ac">

Reply:
I pasted the wrong code in this comment at first. And then I updated it. Did you copy the wrong code I pasted at first?

Reply:
> Have a try to change the path as below

update ~

Reply:
> I pasted the wrong code in this comment at first. And then I updated it. Did you copy the wrong code I pasted at first?

please help to approve

Reply:
Still the same error o(╥﹏╥)o

Reply:
I find the log below in CI build log

`2023-11-30T06:34:52.0611680Z [9034/9035] sccache /usr/bin/c++ -DENABLE_GAPI_PREPROCESSING -DIE_BUILD_POSTFIX=\"\" -DIE_THREAD=IE_THREAD_TBB -DIN_OV_COMPONENT -DOV_BUILD_POSTFIX=\"\" -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"openvino\" -DOV_THREAD=OV_THREAD_TBB -DTBB_PREVIEW_WAITING_FOR_WORKERS=1 -DTEST_PADDLE_MODELS_DIRNAME=\"/__w/openvino/openvino/openvino/bin/intel64/Release/test_model_zoo/paddle_test_models/\"`

TEST_PADDLE_MODELS_DIRNAME should be "./test_model_zoo/paddle_test_models/". Can you add test code in src/frontends/paddle/tests/CMakeLists.txt to print the ${TEST_MODEL_ZOO} and ${TEST_PADDLE_MODELS_DIRNAME} to check their value?

Reply:
> I find the log below in CI build log
> 
> `2023-11-30T06:34:52.0611680Z [9034/9035] sccache /usr/bin/c++ -DENABLE_GAPI_PREPROCESSING -DIE_BUILD_POSTFIX=\"\" -DIE_THREAD=IE_THREAD_TBB -DIN_OV_COMPONENT -DOV_BUILD_POSTFIX=\"\" -DOV_NATIVE_PARENT_PROJECT_ROOT_DIR=\"openvino\" -DOV_THREAD=OV_THREAD_TBB -DTBB_PREVIEW_WAITING_FOR_WORKERS=1 -DTEST_PADDLE_MODELS_DIRNAME=\"/__w/openvino/openvino/openvino/bin/intel64/Release/test_model_zoo/paddle_test_models/\"`
> 
> TEST_PADDLE_MODELS_DIRNAME should be "./test_model_zoo/paddle_test_models/". Can you add test code in src/frontends/paddle/tests/CMakeLists.txt to print the ${TEST_MODEL_ZOO} and ${TEST_PADDLE_MODELS_DIRNAME} to check their value?

I changed it  from a relative path to an absolute path in the previous commit (https://github.com/openvinotoolkit/openvino/pull/20161/commits/8c427ace7cc76c7a81ed9eefa22c66473ee24014), because the run directory in the ci environment is not in the Release directory.

Reply:
No, you should not do that. It should be the relative path. In CI, the test package is copied to another path and run as below

`2023-11-30T06:43:05.9272022Z ##[group]Run source ${INSTALL_DIR}/setupvars.sh
2023-11-30T06:43:05.9273164Z [36;1msource ${INSTALL_DIR}/setupvars.sh[0m
2023-11-30T06:43:05.9274254Z [36;1m${INSTALL_TEST_DIR}/paddle_tests --gtest_print_time=1 \[0m
2023-11-30T06:43:05.9275627Z [36;1m      --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-PaddleTests.xml[0m
2023-11-30T06:43:05.9278296Z shell: bash --noprofile --norc -e -o pipefail {0}
2023-11-30T06:43:05.9278892Z env:
2023-11-30T06:43:05.9279280Z   PIP_CACHE_PATH: /mount/caches/pip/linux
2023-11-30T06:43:05.9279821Z   PYTHON_VERSION: 3.11
2023-11-30T06:43:05.9280314Z   INSTALL_DIR: /__w/openvino/openvino/install
2023-11-30T06:43:05.9281000Z   INSTALL_TEST_DIR: /__w/openvino/openvino/install/tests
2023-11-30T06:43:05.9281619Z ##[endgroup]`

Reply:
> No, you should not do that. It should be the relative path. In CI, the test package is copied to another path and run as below
> 
> `2023-11-30T06:43:05.9272022Z ##[group]Run source ${INSTALL_DIR}/setupvars.sh 2023-11-30T06:43:05.9273164Z �[36;1msource ${INSTALL_DIR}/setupvars.sh�[0m 2023-11-30T06:43:05.9274254Z �[36;1m${INSTALL_TEST_DIR}/paddle_tests --gtest_print_time=1 \�[0m 2023-11-30T06:43:05.9275627Z �[36;1m --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-PaddleTests.xml�[0m 2023-11-30T06:43:05.9278296Z shell: bash --noprofile --norc -e -o pipefail {0} 2023-11-30T06:43:05.9278892Z env: 2023-11-30T06:43:05.9279280Z PIP_CACHE_PATH: /mount/caches/pip/linux 2023-11-30T06:43:05.9279821Z PYTHON_VERSION: 3.11 2023-11-30T06:43:05.9280314Z INSTALL_DIR: /__w/openvino/openvino/install 2023-11-30T06:43:05.9281000Z INSTALL_TEST_DIR: /__w/openvino/openvino/install/tests 2023-11-30T06:43:05.9281619Z ##[endgroup]`

ok, I'll recover it.

Reply:
> 

please help to approve workflows.